### PR TITLE
bugfix/creating financial trasnaction

### PR DESCRIPTION
**Pull request by zetxdev**
fix: creating_financial_transaction #225

### CODE REVIEW

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Comment by dawciobiel** [comment]
- Since the implementation of the service method `createFinancialTransaction` has been changed, all (two) tests dependent on it have been modified. 
- An integration test still needs to be created for this pull request. Only then can the merge be performed. But before this, code review, approval and merge for pull request must first be performed:
https://github.com/bycza-zagroda/track-expenses-app-backend/pull/221

In other words please do not merge this pull request before #221.

**Review by dawciobiel — DISMISSED** [review]
Looks good to me

**Review by dawciobiel — DISMISSED** [review]
Looks good to me

**Review by mirekgab — APPROVED** [review]

**Review by dawciobiel — APPROVED** [review]
Looks good to me
